pymaker/dss.py

```diff
@@ -929,7 +929,7 @@ def rho(self) -> datetime:
     def drip(self) -> Transact:
         return Transact(self, self.web3, self.abi, self.address, self._contract, 'drip', [])
 
-    """Note that join requires use of proxy contract, so join/exit are not provided here."""
+    """Note that join requires use of proxy contract, so join/exit are located in pymaker/Dsr.py"""
```

Comment:
This should be `pymaker/dsrmanager.py`, right?

Reply:
It should be both. Changed to `Join/Exit in Pot can be invoked through pymaker/dsrmanager.py and pymaker/dsr.py`